src/components/AddMovieButton/AddMovieButton.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styles from './AddMovieButton.module.scss';
+import AddMovieModal from '~/components/modals/AddMovieModal';
+
+class AddMovieButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { showAddMovieForm: false };
+
+    this.onAddMovieAction = this.onAddMovieAction.bind(this);
+    this.openAddMovieForm = this.openAddMovieForm.bind(this);
+  }
+
+  onAddMovieAction() {
+    this.setState({ showAddMovieForm: false });
+  }
```

Comment:
```js
onAddMovieAction = () => {
    this.setState({ showAddMovieForm: false });
}
```

Reply:
Done. But had to add @babel/plugin-proposal-class-properties and appropriate linting